packages/bundle/src/__tests__/createDirectLine.spec.js

```diff
@@ -0,0 +1,30 @@
+jest.mock('botframework-directlinejs', () => {
+  class DirectLine {
+    constructor({ botAgent }) {
+      this.botAgent = botAgent;
+    }
+  }
+
+  return { DirectLine };
+});
+
+import * as WebChatFull from '../index';
+import * as WebChatMinimal from '../index-minimal';
+import * as WebChatES5 from '../index-es5';
+
+describe('Web Chat createDirectLine', () => {
+  it('Full', () => {
```

Comment:
The "describe()" + "it()" should form a meaningful sentence, i.e. one-line summary. By reading just this sentence, the reader know what is being tested, and what is expected. (Some of our current test don't pass this test.)

```suggestion
  it('for full bundle', () => {
```